iOS/G3MiOSSDK/Commons/Rendererers/TileLoDTester.hpp

```diff
@@ -0,0 +1,70 @@
+//
+//  TileLoDTester.hpp
+//  G3MiOSSDK
+//
+//  Created by Jose Miguel SN on 4/12/15.
+//
+//
+
+#ifndef TileLoDTester_hpp
+#define TileLoDTester_hpp
+
+#include "TileTessellator.hpp"
+#include "TileTexturizer.hpp"
+#include "Context.hpp"
+
+class TileLoDTesterData{
+  //Empty class. Each TileLoDTester will implement a different set of associated data and will
+  //store it inside the tile using its unique level id
+public:
+#ifdef C_CODE
+  virtual ~TileLoDTesterData() { }
+#endif
+#ifdef JAVA_CODE
+  void dispose(){}
+#endif
+};
+
+class TileLoDTester{
+  
+  TileLoDTester* _nextTesterRightLoD;
+  TileLoDTester* _nextTesterWrongLoD;
+  TileLoDTester* _nextTesterVisible;
+  TileLoDTester* _nextTesterNotVisible;
+  
```

Comment:
I don't think those attributes have to be here.   This abstract class can't assume the shape of the subclasses.